libsemigroups_pybind11/adapters.py

```diff
@@ -6,114 +6,152 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 
-# pylint:disable=no-name-in-module, unused-import
-
 """
 This package provides the user-facing python part of libsemigroups_pybind11 for
 various adapters from libsemigroups.
 """
 
-from typing import Any
-
-from typing_extensions import Self
+from typing import TypeVar as _TypeVar
+from typing_extensions import Self as _Self
 
-from _libsemigroups_pybind11 import (
-    ImageRightActionBMat8BMat8 as _ImageRightActionBMat8BMat8,
-    ImageLeftActionBMat8BMat8 as _ImageLeftActionBMat8BMat8,
-    ImageRightActionPPerm1PPerm1 as _ImageRightActionPPerm1PPerm1,
-    ImageLeftActionPPerm1PPerm1 as _ImageLeftActionPPerm1PPerm1,
-    ImageRightActionPPerm1List as _ImageRightActionPPerm1List,
+from _libsemigroups_pybind11 import (  # pylint: disable=no-name-in-module,unused-import
```

Comment:
```suggestion
from _libsemigroups_pybind11 import (  # pylint: disable=no-name-in-module
```
since we do use all of the imports.